apps/docs/content/guides/platform/access-control.mdx

```diff
@@ -1,85 +1,258 @@
 ---
 title: 'Access Control'
-description: 'Roles and permissions at the organization level'
+description: 'Roles and permissions at the organization and project levels'
 ---
 
-Supabase provides granular access controls to manage permissions across your organizations.
-For each organization, a member can have one of the following roles:
+Supabase provides granular access controls to manage permissions across your organizations and projects.
 
-- Owner
-- Administrator
-- Developer
+For each organization and project, a member can have one of the following roles:
+
+- **Owner**: full access to everything in organization and project resources.
+- **Administrator**: full access to everything in organization and project resources except updating organization settings, transferring projects outside of the organization, and adding new owners.
+- **Developer**: read-only access to organization resources and content access to project resources but cannot change any project settings.
+- **Read-Only**: read-only access to organization and project resources.
 
 <Admonition type="note">
 
-Additional roles (Read-only and Billing) are available on the [team and enterprise plans](https://supabase.com/pricing).
+Owner, Administrator, and Developer roles are available on all projects but roles, such as Read-Only, are only available on the [Team and Enterprise plans](https://supabase.com/pricing).
```

Comment:
This was so hard to read first time I read it. 
Can it not just say 

```suggestion
Read-only and Billing roles only available on [team and enterprise plans](https://supabase.com/pricing).

```

Reply:
good suggestion, will do

Reply:
we don't want to mention Billing role yet